frontend_tests/node_tests/stream_data.js

```diff
@@ -599,3 +599,37 @@ zrequire('marked', 'third/marked/lib/marked');
     assert.equal(antarctica_sub.color, '#76ce90');
 }());
 
+(function test_initialize_from_page_params() {
+    function initialize() {
+        page_params.subscriptions = [{
+            name: 'subscriptions'
+        }];
+    
+        page_params.unsubscribed = [{
+            name: 'unsubscribed'
+        }];
+    
+        page_params.never_subscribed = [{
+            name: 'never_subscribed'
+        }];
+    }
+
+    initialize();
+    page_params.realm_notifications_stream_id = -1;
+    stream_data.initialize_from_page_params();
+    assert(!page_params.subscriptions);
```

Comment:
Can you put a line break before this line? It's easier to read if you put the asserts apart from the set-up code.